test/test_gpu3.cpp

```diff
@@ -8664,6 +8664,31 @@ TEST_F(NVFuserTest, Reduction3DConstantIterationDomain) {
       executor_cache.fusion(), cg_outputs, inputs, {ref}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, FusionTensorRankLimit) {
```

Comment:
Can we move this test to a different file? `test_gpu\d.cpp` are already enormous. It's better to only remove tests from them.